src/pipx/main.py

```diff
@@ -486,6 +488,12 @@ def _add_upgrade(subparsers, venv_completer: VenvCompleter, shared_parser: argpa
         help="Modify existing virtual environment and files in PIPX_BIN_DIR and PIPX_MAN_DIR",
     )
     add_pip_venv_args(p)
+    p.add_argument(
```

Comment:
I think you also have to run `add_python_options(p)` to add the `--python` flag.

Reply:
Gotcha, sorry I missed that last night